licenses/management/commands/publish.py

```diff
@@ -131,7 +131,7 @@ def run_django_distill(self):
 
         self.stdout.write(f"\n{hostname}:{output_dir}")
         save_url_as_static_file(
-            output_dir, reverse("metadata"), "licenses/metadata.yaml"
+            output_dir, reverse("metadata"), "licenses/metadata.yaml", False
```

Comment:
Example with named arguments for readability.

```suggestion
            output_dir,
            url=reverse("metadata"),
            relpath="licenses/metadata.yaml",
            html=False
```